Content.Shared/Foldable/FoldableComponent.cs

```diff
@@ -16,9 +16,17 @@ public sealed partial class FoldableComponent : Component
 {
     [DataField("folded")]
     public bool IsFolded = false;
+
+    [DataField]
+    public bool FitsInContainerUnfolded = false;
+
+    [DataField]
+    public bool CanFoldInsideContainer = false;
+
+    [DataField]
+    public bool CanFoldWhileEquipped = false;
 }
```

Comment:
Just make it container folding it doesn't need 3 bools.

Reply:
Derp, I should've realized that the first two are the same thing.
I split out the "equipped" logic into FoldAttemptEvent, since I'd like to use this logic to allow flipping softcaps while they're on your head. (but still disallow folding bandanas, since they go into different slots upon folding.)